pkg/codegen/nodejs/gen.go

```diff
@@ -987,6 +993,7 @@ func (mod *modContext) sdkImports(nested, utilities bool) []string {
 	if nested {
 		imports = append(imports, fmt.Sprintf("import * as inputs from \"%s/types/input\";", relRoot))
 		imports = append(imports, fmt.Sprintf("import * as outputs from \"%s/types/output\";", relRoot))
+		imports = append(imports, fmt.Sprintf("import * as enums from \"%s/types/enums\";", relRoot))
```

Comment:
Should this import be conditional based on whether or not any enums are present? Probably not a big deal.